source/extensions/filters/http/jwt_authn/token_cache.cc

```diff
@@ -0,0 +1,30 @@
+#include "extensions/filters/http/jwt_authn/token_cache.h"
+
+using std::chrono::system_clock;
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace JwtAuthn {
+
+TokenCache::TokenCache(int cache_size)
+    : SimpleLRUCache<std::string, ::google::jwt_verify::Jwt>(cache_size) {}
+
+TokenCache::~TokenCache() { clear(); }
+
+bool TokenCache::lookupTokenCache(const std::string& token) {
```

Comment:
Just call "find"  or "lookup". 

Hmm,  if only return bool.  How do you use cached value?

Reply:
Done.